src/components/login/Login.tsx

```diff
@@ -25,16 +25,16 @@ const Login = () => {
   const dispatch = useDispatch();
   const router = useRouter();
   const boundAuthActions = bindActionCreators(authActions, dispatch);
+  const [isSubmitted, setIsSubmitted] = useState(false);
```

Comment:
I think that this might be redundant. Formik is already handling isSubmitted flag.